Acd 843 (#63)
Actindo-AG/Documentation: Payments/UserInterface/07_Logging.md

# Logging

> The Logging menu entry is used to check different logs.

Reviewer: ```suggestion
The *Logging* menu entry is used to check different logs.
```

Author: done

It only contains the [LIST](07a_ListLogging.md) tab.

> >[Info] The Logging menu entry provides an overall view on all logs that has been created during the different phases of the payment processing. It is recommended to open the LIST sub-tabs of the Authorizations, Refunds etc. menu entries to have a more specific view on the logs. For example, see [Authorizations &ndash; Logs](01a_ListAuthorizations.md#authorizations-–-logs).

Reviewer: ```suggestion
> [Info] The *Logging* menu entry provides an overall view on all logs that have been created during the different phases of the payment processing. It is recommended to open the *LIST* sub-tabs of the corresponding menu entry to have a more specific view on the logs, such as [Authorizations &ndash; Logs](01a_ListAuthorizations.md#authorizations-–-logs).
```

Author: Done
